tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts

```diff
@@ -588,6 +613,23 @@ export class ScalarCardContainer implements CardRenderer, OnInit, OnDestroy {
     );
 
     this.hparamsEnabled$ = this.store.select(getEnableHparamsInTimeSeries);
+
+    this.runToHparams$ = this.store.select(getDashboardRunsToHparams).pipe(
+      map((runToHparamsAndMetrics: RunToHparamsAndMetrics): RunToHparams => {
+        const runToHparams: RunToHparams = {};
+        for (const [runName, {hparams}] of Object.entries(
+          runToHparamsAndMetrics
+        )) {
+          runToHparams[runName] = {};
+          for (const {name: hparamName, value} of hparams) {
+            runToHparams[runName][hparamName] = value;
+          }
+        }
+        return runToHparams;
+      })
+    );
```

Comment:
If you make this a selector it will be easier to test. It will also be easier to share if we ever end up wanting to compute this someplace else.

Reply:
I was actually debating whether to do this. Replaced this with a new runs selector called `getRunToHparamMap`

Reply:
Thank you!